### PR TITLE
Seo default image

### PR DESCRIPTION
**Pull request by GaryPWhite**
Set up default seo image for sharing wayfair.github.io links

### CODE REVIEW

**Comment by GaryPWhite** [comment]
confirmed with slack that this works! Just needs a review :)

File: gatsby-config.js [hunk 1]
```diff
@@ -2,18 +2,19 @@ module.exports = {
...
-    siteUrl: `http://wayfair.github.io`,
+    author: `opensource@wayfair.com`,
+    siteUrl: `https://wayfair.github.io`,
+    keywords: ["opensource", "wayfair", "Wayfair", "OSS"],
   },
   plugins: [
     `gatsby-plugin-sitemap`,
     {
-      resolve: 'gatsby-plugin-robots-txt',
+      resolve: "gatsby-plugin-robots-txt",
```
Comment (draco2003) [inline gatsby-config.js:12]: Non-blocking Nit: Seems like we are switching from Single Quotes to Double quotes here, while the majority of the rest of the config uses back ticks? any reason not to consistently use back ticks?

**Review by draco2003 — APPROVED** [review]

File: gatsby-config.js (hunk 1 above)
Comment (GaryPWhite) [reply gatsby-config.js:12]: nope just what I did by habit

File: gatsby-config.js [hunk 2]
```diff
@@ -2,30 +2,31 @@ module.exports = {
   siteMetadata: {
     title: `Wayfair Open Source`,
     description: `Home of the Open Source Program Office and Wayfair specific documenting for working with OSS`,
-    author: ``,
-    siteUrl: `http://wayfair.github.io`,
+    author: `opensource@wayfair.com`,
+    siteUrl: `https://wayfair.github.io`,
+    keywords: [`opensource`, `wayfair`, `Wayfair`, `OSS`],
```
Comment (lelia) [inline gatsby-config.js:7]: does it make sense to also include `OSPO` here?

Comment (GaryPWhite) [reply gatsby-config.js]: donezo

Comment (lelia) [reply gatsby-config.js]: yamzo

**Review by lelia — APPROVED** [review]

**Pull request merged**